data/operating_systems/maintained.json

```
@@ -192,5 +192,11 @@
       "description": "penetration testing and security lab",
       "website": "http://www.wifislax.com/"
     }
+    {
+      "name": "Ziion",
+      "base": "Ubuntu",
+      "description": "blockchain security testing and development solution",
```

Format convention https://inventory.raw.pm/docs/format

> Description begins with an uppercase letter and doesn't end with a dot (.)
```suggestion
      "description": "Blockchain security testing and development VM",
```